rasa/shared/core/training_data/story_reader/yaml_story_reader.py

```diff
@@ -514,6 +525,10 @@ def _parse_action(self, step: Dict[Text, Any]) -> None:
 
         self._add_event(action_name, {})
 
+    def _parse_bot_message(self, step: Dict[Text, Any]) -> None:
+        bot_message = step.get(KEY_BOT_END_TO_END_MESSAGE, "")
+        self._add_event("", {"action_text": bot_message})
```

Comment:
```suggestion
        self._add_event("", {ACTION_TEXT: bot_message})
```
from `rasa/shared/nlu/constants.py`